bbb-graphql-middleware/internal/hascli/conn/writer/writer.go

```diff
@@ -74,7 +83,7 @@ RangeLoop:
 
 				if fromBrowserMessageAsMap["type"] == "stop" {
 					var queryId = fromBrowserMessageAsMap["id"].(string)
-					if browserConnection.JsonPatchSupported {
+					if browserConnection.ActiveSubscriptions[queryId].JsonPatchSupported {
```

Comment:
As you are accessing the map by a key, ensure it's not concurrently modified (use mutex around it)